iron-overlay-behavior.html

```diff
@@ -296,14 +312,14 @@
     _prepareRenderOpened: function() {
       this._manager.addOverlay(this);
 
-      if (this.withBackdrop) {
-        this.backdropElement.prepare();
-        this._manager.trackBackdrop(this);
-      }
-
       this._preparePositioning();
       this.fit();
       this._finishPositioning();
```

Comment:
these lines about positioning will update the element's style, which in some cases influences the calculation of `z-index`. Executing these before the backdrop updates fixes the issue.